core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -328,12 +342,13 @@ var StoryEditorPage = function() {
       'ExplorationIdInput takes too long to be visible'
```

Comment:
Above in `this.expectSaveStoryDisabled`, can we wait for `saveStoryButton` to be visible before getting its attribute?